frontend/website/src/components/Leaderboard.re

```diff
@@ -214,7 +221,9 @@ let make = () => {
     <div id="testnet-leaderboard" className=Styles.leaderboard>
       <div className=Styles.headerRow>
         <span className=Styles.flexEnd> {React.string("Rank")} </span>
-        <span> {React.string("Name")} </span>
+        <span style={ReactDOMRe.Style.make(~gridColumn="3 / 4", ())}>
```

Comment:
You can just use inline bs-css styles here instead of mixing the two. I dunno if this will work exactly but something like:
```suggestion
        <span style={Css.(style([gridColumn(3/4)]))}>
```

Reply:
good catch on this. I struggled with this problem before and didn't realize this was a solution. Thanks!

Reply:
will add to frontend library tips!